.github/labeler.yml

```diff
@@ -52,24 +52,106 @@ area/docs:
   - diagrams/*
   - diagrams/**/*
 
+# -----------------collectors----------------------
+
 area/collectors:
   - collectors/*
   - collectors/**/*
 
-area/ebpf:
-  - collectors/ebpf.plugin/*
-  - collectors/ebpf.plugin/**/*
+collectors/plugins.d:
+  - collectors/plugins.d/*
+  - collectors/plugins.d/**/*
 
-area/external/python:
-  - collectors/python.d.plugin/*
-  - collectors/python.d.plugin/**/*
+collectors/apps:
+  - collectors/apps.plugin/*
+  - collectors/apps.plugin/**/*
+
+collectors/cgroups:
+  - collectors/cgroups.plugin/*
+  - collectors/cgroups.plugin/**/*
 
-area/external:
+collectors/charts.d:
   - collectors/charts.d.plugin/*
   - collectors/charts.d.plugin/**/*
+
```

Comment:
Please, add [checks](https://github.com/netdata/netdata/tree/master/collectors/checks.plugin).

Reply:
I don't think we need it. It is not supposed to be used by users.

Reply:
This would be the unique collector not listed in this file. 
Let us reach a consensus, @vlvkobal what is your opinion about this?

Reply:
I agree with Ilya. It's our internal instrumentation.